code/controllers/subsystems/jobs.dm

```diff
@@ -122,7 +115,7 @@ SUBSYSTEM_DEF(jobs)
 	. = ..()
 
 /datum/controller/subsystem/jobs/proc/guest_jobbans(var/job)
-	for(var/dept in list(COM, MSC, SEC))
+	for(var/dept in list("command", "misc", "security"))
```

Comment:
These string IDs should really be `#define`d somewhere rather than used as barestrings, if only for compile-time testing.

Reply:
I've now removed the majority of hard coded references.